lib/filter_nb.c

```diff
@@ -574,10 +564,10 @@ static int lib_access_list_entry_destination_network_destroy(
 
 	f = nb_running_get_entry(args->dnode, NULL, true);
 	fc = &f->u.cfilter;
-	fc->extended = 0;
-	cisco_unset_addr_mask(&fc->mask, &fc->mask_mask);
+	fc->extended = 1;
```

Comment:
This is not an extended case right?

Reply:
Yes it is, you might be confusing with the old `destroy` function that was here.